src/integration-tests/introspector/introspectTest.sh

```diff
@@ -99,6 +99,21 @@ export ALLOW_DYNAMIC_CLUSTER_IN_FMW=${ALLOW_DYNAMIC_CLUSTER_IN_FMW:-false}
 # whether this test run is expecting a domain validation error
 export EXPECT_INVALID_DOMAIN=${EXPECT_INVALID_DOMAIN:-false}
 
+DOMAIN_SOURCE_TYPE=${DOMAIN_SOURCE_TYPE:-Image}
```

Comment:
This should be 'PersistentVolume'.  It's interesting that 'Image' works.  The test does not create an Image in its default path.